Projeto-Stream/src/router/index.ts

```diff
@@ -20,6 +20,11 @@ const router = createRouter({
       path: '/movie',
       name: 'movie',
       component: MovieView
+    },
+    {
+        path: "/details/:type/:id",
+        name: "details",
+        component: ()=> import("../views/Home/homeDetail.vue")
```

Comment:
Change double quotes (" ") to single quotes (' ') or single quotes to double quotes.
Seguir um padrão para aspas no projeto.